utilities/transactions/write_unprepared_txn.h

```diff
@@ -40,13 +40,15 @@ class WriteUnpreparedTxnReadCallback : public ReadCallback {
     // behind reseek optimizations are no longer valid.
   }
 
-  // TODO(myabandeh): override Refresh when Iterator::Refresh is supported
+  void Refresh(SequenceNumber seq) override {
+    max_visible_seq_ = std::max(max_visible_seq_, seq);
+    wup_snapshot_ = seq;
+  }
+
  private:
   SequenceNumber CalcMaxVisibleSeq(WriteUnpreparedTxn* txn,
                                    SequenceNumber snapshot_seq) {
     SequenceNumber max_unprepared = CalcMaxUnpreparedSequenceNumber(txn);
-    assert(snapshot_seq < max_unprepared || max_unprepared == 0 ||
```

Comment:
Instead of removing the assert, I think we can just add `|| snapshot_seq  == 0`?

Reply:
We have the case where the ::Get did not specify the snapshot and the latest seq is picked as its snapshot seq. In that case, the snapshot_seq would be larger than max_unprepared.